### PR TITLE
Asher west nile virus

### PR DESCRIPTION
**Pull request by asherps**

### CODE REVIEW

File: .DS_Store
Comment (simonleandergrimm) [inline .DS_Store:1]: Shouldn't be here.

File: .vscode/settings.json
Comment (simonleandergrimm) [inline .vscode/settings.json:1]: Shouldn't be here.

File: Titanic.ipynb
Comment (simonleandergrimm) [inline Titanic.ipynb:1]: Shouldn't be here.

File: pathogens/west_nile_virus.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+from pathogen_properties import *
+
+# ***We should add a CI once we know how to get from tested individuals to CI
+
+background = """“West Nile virus (WNV) is the leading cause of mosquito-borne 
+disease in the continental United States.  It is most commonly spread to 
+people by the bite of an infected mosquito. Cases of WNV occur during mosquito 
+season, which starts in the summer and continues through fall. There are no 
+vaccines to prevent or medications to treat WNV in people. Fortunately, most 
+people infected with WNV do not feel sick. About 1 in 5 people who are
```
Comment (simonleandergrimm) [inline pathogens/west_nile_virus.py:10]: I would cut "Fortunately, most people infected with WNV do not feel sick"

File: pathogens/west_nile_virus.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+pathogen_chars = PathogenChars(
+    na_type=NAType.RNA,
+    enveloped=Enveloped.ENVELOPED,
+    taxid=TaxID(11082),
+)
+
+LA_county_cases_in_2020 = IncidenceAbsolute(
+    annual_infections=90,
+    tag="LA-2020",
+    source="https://westnile.ca.gov/pdfs/VBDSAnnualReport20.pdf#?page=23",
```
Comment (simonleandergrimm) [inline pathogens/west_nile_virus.py:23]: Great source! Please add the date here `2020`.

File: pathogens/west_nile_virus.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+LA_county_population = Population(
+    people=10_014_009,
+    tag="LA-2020",
+    source="https://www.census.gov/quickfacts/fact/table/losangelescountycalifornia,CA/POP010220#POP010220",
+)
+
+
+west_nile_duration = SheddingDuration(
+    days=7,
```
Comment (simonleandergrimm) [inline pathogens/west_nile_virus.py:34]: It doesn't say 7 days here, but 3 to 6 days (though in the following sentence they mention that illnesses sometimes last a lot longer). There are also no sources in the linked page; there should be better literature on this./

File: pathogens/west_nile_virus.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    source="https://myhealth.alberta.ca/Health/aftercareinformation/pages/conditions.aspx?hwid=abo5809#:~:text=In%20mild%20cases%20of%20West%20Nile%2C%20symptoms%20usually%20last%20for%203%20to%206%20days%2C%20and%20you%20can%20recover%20at%20home.%20If%20you%20get%20a%20more%20severe%20case%20of%20West%20Nile%2C%20symptoms%20can%20last%20for%20weeks%20or%20months%2C%20and%20you%20may%20need%20to%20stay%20in%20the%20hospital%20so%20you%20can%20get%20medicine%20to%20help%20you%20recover.",
+)
+
+# 4/5 of people with WNV are asymptomatic, and therefore probably did not get
+# tested
+asymptomatic_multiplier = Scalar(
```
Comment (simonleandergrimm) [inline pathogens/west_nile_virus.py:40]: Nice, that's a good catch.

Comment (simonleandergrimm) [reply pathogens/west_nile_virus.py:40]: Please use this source instead: https://www.cdc.gov/mmwr/volumes/70/ss/ss7001a1.htm#:~:text=An%20estimated%2070%25%E2%80%9380%25%20of%20WNV%20infections%20are%20asymptomatic%20(8%2C9), and briefly check if the primary sources are reasonable (they sometimes really aren't)

**Review by simonleandergrimm — CHANGES_REQUESTED** [review]
Overall this looks very good, nice job. I added some comments for things to change.

File: .vscode/settings.json
Comment (simonleandergrimm) [inline .vscode/settings.json:1]: This shouldn't be here

File: .DS_Store
Comment (simonleandergrimm) [inline .DS_Store:1]: This shouldn't be here

File: pathogens/west_nile_virus.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
+from pathogen_properties import *
+
+# ***We should add a CI once we know how to get from tested individuals to CI
```
Comment (simonleandergrimm) [inline pathogens/west_nile_virus.py:3]: Is this a TODO? If so you can add TODO before the comment.

File: pathogens/west_nile_virus.py [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+LA_county_cases_in_2020 = IncidenceAbsolute(
+    annual_infections=90,
+    tag="LA-2020",
+    date="2020",
+    source="https://westnile.ca.gov/pdfs/VBDSAnnualReport20.pdf#?page=23",
+)
+
+LA_county_population = Population(
+    people=10_014_009,
+    tag="LA-2020",
```
Comment (simonleandergrimm) [inline pathogens/west_nile_virus.py:28]: Also add date here, missed this in prior review

File: pathogens/west_nile_virus.py [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+    source="https://www.census.gov/quickfacts/fact/table/losangelescountycalifornia,CA/POP010220#POP010220",
+)
+
+
+west_nile_duration = SheddingDuration(
```
Comment (simonleandergrimm) [inline pathogens/west_nile_virus.py:33]: I would use this source, if we go by trustworthiness alone the European CDC is better than the Albert Health Department. They give 2-18, so you can give 10 as `days` and 2-18 as the `confidence_interval` https://www.ecdc.europa.eu/en/west-nile-fever/facts#:~:text=West%20Nile%20virus%20infection%20can%20be%20confirmed%20by%20virus%20genome%20detection%20or%20virus%20isolation.%20Viral%20genome%20is%20typically%20detectable%20in%20plasma%20from%202%E2%80%9318%20days%20post%2Dinfection

**Review by simonleandergrimm — APPROVED** [review]
Nice! This is nearly ready. After adapting the proposed changes, you can add Jeff as a reviewer and make him an assignee!

File: pathogens/west_nile_virus.py (hunk 7 above)
Comment (asherps) [reply pathogens/west_nile_virus.py:33]: Thanks! Should this count as a confidence interval? In some ways, it might be more of a range for our purposes?

**Comment by jeffkaufman** [comment]
It looks like this PR was moved when tests were failing?  We're now getting:

```
ERROR: test_properties_exist (__main__.TestPathogens.test_properties_exist) (pathogen='west_nile_virus')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jeffkaufman/code/p2ra/./test.py", line 42, in test_properties_exist
    for estimate in pathogen.estimate_prevalences():
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jeffkaufman/code/p2ra/pathogens/west_nile_virus.py", line 56, in estimate_prevalences
    .scale(asymptomatic_multiplier)
     ^^^^^
AttributeError: 'Prevalence' object has no attribute 'scale'
```

I'll send a PR to fix

**Comment by jeffkaufman** [comment]
Actually, this is so small that I just pushed 19a908b8502898b076d3f6066dcaababa01ec35d without review

**Comment by jeffkaufman** [comment]
> add Jeff as a reviewer and make him an assignee!

It looks like instead this was just merged?

**Pull request merged**